Dockerfile

```diff
@@ -13,15 +13,19 @@ RUN apt-get update \
 	&& apt-get install -qqy --no-install-recommends\
         # This is for enabling the program to be run with watch
         procps \
-        wkhtmltopdf \
         # Required to run PDF generation
         xvfb \
         xauth \
+        wget apt-utils libjpeg62-turbo libxrender1 xfonts-75dpi xfonts-base fontconfig \
 	&& rm -rf /var/lib/apt/lists/*
 
+RUN cd /root && \
+    wget https://downloads.wkhtmltopdf.org/0.12/0.12.5/wkhtmltox_0.12.5-1.stretch_amd64.deb && \
+    dpkg -i wkhtmltox_0.12.5-1.stretch_amd64.deb
+
 # Wrap pdf creation in a xvfb-run to enable headless pdf creation in the container
-RUN echo "#!/bin/bash\nxvfb-run $(which wkhtmltopdf) \"\$@\"" >> /usr/local/bin/wkhtmltopdf \
-    && chmod +x /usr/local/bin/wkhtmltopdf
+# RUN echo "#!/bin/bash\nxvfb-run $(which wkhtmltopdf) \"\$@\"" >> /usr/local/bin/wkhtmltopdf \

```

Comment:
If this and `xvfb` are not needed anymore, let's remove them. No need to leave that stuff in here.